src/modules/tags/TagDetail.js

```diff
@@ -39,6 +46,29 @@ const TagDetail = (props) => {
     }
   };
 
+  const renderSpotFeatureItem = (spotFeature) => {
+    const spot = useSpots.getSpotById(spotFeature.spotId);
+    if (!isEmpty(spot)) {
+      return (
+        <ListItem
+          containerStyle={commonStyles.listItem}
+          onPress={() => props.openSpot(spot)}
```

Comment:
This should go directly to the feature detail.

Reply:
needs discussion.

Reply:
Similar to the discussion we had about fetching the FeatureItem component, where we do not know which feature item component to fetch, here also, we have list of features and we do not know which feature detail page to go to.